.nvmrc

```diff
@@ -1 +1 @@
-8.14.0
+14.15.1
```

Comment:
I see our other repos using 14.15.0 -- was there a driver for using 14.15.1?

Can you update [this line](https://github.com/spanishdict/pn-logging/blob/master/package.json#L6) to be `>=10`? That's the version of node that introduced isNativeError.

Reply:
Ah, I didn't realize others were on 14.15.0, I just used what I had downloaded, so I will update to match